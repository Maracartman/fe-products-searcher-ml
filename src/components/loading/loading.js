import React from 'react';
import Spinner from 'react-bootstrap/Spinner';
import Row from 'react-bootstrap/Row';
import Col from 'react-bootstrap/Col';
import './loading.scss';

export const Loading = () => (
  <Row>
    <Col className={'spinner__container'}>
      <Spinner role="status" animation="border" variant="warning" />
    </Col>
  </Row>
);
