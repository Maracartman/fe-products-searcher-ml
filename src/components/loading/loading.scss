.spinner__container {
    min-height: 400px;

    .text-warning {
        position: relative;
        display: block;
        margin: 200px auto;
    }
}



// Small devices (landscape phones, 576px and up)
@media (min-width: 576px) {
    .spinner__container {
        min-height: 400px;
    }
}

// Medium devices (tablets, 768px and up)
@media (min-width: 768px) {
    .spinner__container {
        min-height: 400px;
    }
}

// Large devices (desktops, 992px and up)
@media (min-width: 992px) {
    .spinner__container {
        min-height: 400px;
    }
}

// Extra large devices (large desktops, 1200px and up)
@media (min-width: 1200px) {
    .spinner__container {
        min-height: 400px;
    }
}