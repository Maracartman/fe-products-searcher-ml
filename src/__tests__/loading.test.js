import React from 'react';
import { shallow } from 'enzyme';
import { Loading } from '../components/loading/loading';

describe('Loading test suite', () => {
  it('Should loading match Snapshot', () => {
    const loading = shallow(<Loading />);
    expect(loading).toMatchSnapshot();
  });
});
