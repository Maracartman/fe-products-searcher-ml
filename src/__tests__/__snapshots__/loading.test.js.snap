// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading test suite Should loading match Snapshot 1`] = `ShallowWrapper {}`;
